### ISSUE TITLE
Consider renaming the "Site Settings" shortcut

### ISSUE THREAD

**Issue by andreban** [GoogleChrome/android-browser-helper#181]
As an application downloaded from the Play Store, the "Site Settings" may not be clear for users. Consider renaming to "Application Settings" or something similar.

More context:
- https://twitter.com/tomayac/status/1306840186219892736
- https://twitter.com/firt/status/1307014143317364736

@padr31 FYI

**Comment by FluorescentHallucinogen** [comment]
Wouldn't it be better to add the ability to customize the name and icon?

See the rationale: https://twitter.com/daviddalbusco/status/1307014586743361536

**Comment by FluorescentHallucinogen** [comment]
@b1tr0t cc.

**Comment by andreban** [comment]
Adding the image from the tweet @FluorescentHallucinogen mentioned. The icon is looking very off in this image. Another point to consider here is internationalisation, which doesn't seem to be supported in the code.

![EiNyDG9XgAM39ef](https://user-images.githubusercontent.com/1733592/93636024-1b5e0a00-f9eb-11ea-997d-07ac3b5cc35a.jpeg)

**Comment by peterpeterparker** [comment]
Thank you @FluorescentHallucinogen 👍 

As I meant in my tweet and  above screenshot, I would really appreciate the ability to style the icon (not just for design reason but also for accessibility reason) and I also personally would rather like to display "App Settings"  instead of "Site Settings" but, that's a personal point of view of course.

Regarding i18n, definitely would be nice. I think, or if I remember correctly, there is already an ongoing topic regarding the weblinks as currently you can't translate these neither.

**Comment by gstepniewski-google** [comment]
Closing due to no updates in 5+ years (and we're currently working on adding string translations)

**Comment by peterpeterparker** [comment]
Yeah, after 5+ years, I couldn't care less about it anymore. Good call to close this.
